src/main/java/org/scijava/nativelib/NativeLibraryUtil.java

```diff
@@ -77,11 +77,11 @@
 
 	public static enum Architecture {
 		UNKNOWN, LINUX_32, LINUX_64, LINUX_ARM, LINUX_ARM64, WINDOWS_32, WINDOWS_64, OSX_32,
-			OSX_64, OSX_PPC
+			OSX_64, OSX_PPC, AIX_32, AIX_64
 	}
 
 	private static enum Processor {
-		UNKNOWN, INTEL_32, INTEL_64, PPC, ARM, AARCH_64
+		UNKNOWN, INTEL_32, INTEL_64, PPC, PPC_64, ARM, AARCH_64
```

Comment:
Added 64 bit version of PowerPC processor. On 64 bit AIX os.arch parameter returns ppc64.